RecoMuon/L3MuonProducer/python/l3MuonCandidateProducerFromMuons_cfi.py

```diff
@@ -1,5 +1,6 @@
 import FWCore.ParameterSet.Config as cms
 
```

Comment:
This `cfi` file can now be removed. I don't see it used anywhere, and it will now be created automatically via `fillDescriptions`.